Clarify: "Received error for watch, will try to reconnect"

Stumbled upon this one:

```
2021-10-05 14:32:49,815 WARN  [org.bf2.cos.fleetshard.operator.connector.ConnectorOperatorEventSource] (OkHttp https://172.21.0.1/...) Received error for watch, will try to reconnect.: io.fabric8.kubernetes.client.WatcherException: too old resource version: 30632126 (48104904)
	at io.fabric8.kubernetes.client.dsl.internal.AbstractWatchManager.onStatus(AbstractWatchManager.java:273)
	at io.fabric8.kubernetes.client.dsl.internal.AbstractWatchManager.onMessage(AbstractWatchManager.java:257)
	at io.fabric8.kubernetes.client.dsl.internal.WatcherWebSocketListener.onMessage(WatcherWebSocketListener.java:93)
	at okhttp3.internal.ws.RealWebSocket.onReadMessage(RealWebSocket.java:322)
	at okhttp3.internal.ws.WebSocketReader.readMessageFrame(WebSocketReader.java:219)
	at okhttp3.internal.ws.WebSocketReader.processNextFrame(WebSocketReader.java:105)
	at okhttp3.internal.ws.RealWebSocket.loopReader(RealWebSocket.java:273)
	at okhttp3.internal.ws.RealWebSocket$1.onResponse(RealWebSocket.java:209)
	at okhttp3.RealCall$AsyncCall.execute(RealCall.java:174)
	at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: too old resource version: 30632126 (48104904)
	... 13 more

```

Is it something to be expected?

--- a/cos-fleetshard-support/src/main/java/org/bf2/cos/fleetshard/support/watch/AbstractWatcher.java
+++ b/cos-fleetshard-support/src/main/java/org/bf2/cos/fleetshard/support/watch/AbstractWatcher.java
@@ -53,7 +53,7 @@ public abstract class AbstractWatcher<T> implements Watcher<T>, AutoCloseable {
             return;
         }
         if (e.isHttpGone()) {
-            logger.warn("Received error for watch, will try to reconnect.", e);
+            logger.info("Received recoverable error for watch, will try to reconnect (message: {})", e.getMessage());
             close();
             start();
         } else {

